matter_server/server/device_controller.py

```diff
@@ -567,6 +578,18 @@ async def send_device_command(
         cluster_cls: Cluster = ALL_CLUSTERS[cluster_id]
         command_cls = getattr(cluster_cls.Commands, command_name)
         command = dataclass_from_dict(command_cls, payload, allow_sdk_types=True)
+        if node_id >= TEST_NODE_START:
+            LOGGER.info(
```

Comment:
Aren't these logging messages more appropriate at debug level?

Reply:
Well it depends on the usecase. In this case the logging is a replacement of the action and not logging the action, so I figured INFO more useful.

This just logs all actions performed on a test node instead of actually performing it.

Just some thoughts. I can change it to debug, no problem.